soccer/gameplay/GameplayModule.hpp

```diff
@@ -120,6 +124,9 @@ class GameplayModule {
     /// This is held while plays are running.
     QMutex _mutex;
 
+    static ConfigDouble* inset;
+    double oldInset;
```

Comment:
Also, since it isn't necessarily clear to someone reading this header file what `inset` refers to, maybe a better name would be `fieldEdgeInset` to give it a little more context.